packages/ember-cli-code-coverage/lib/attach-middleware.js

```diff
@@ -134,17 +137,22 @@ function adjustCoverage(coverage, options) {
       namespaceMappings,
       modifyAssetLocation
     );
-    coverage[filePath].path = path.relative(root, relativeToProjectRoot);
-    memo[path.relative(root, relativeToProjectRoot)] = coverage[filePath];
+    coverage[filePath].data.path = path.relative(root, relativeToProjectRoot);
```

Comment:
@vstefanovic97 out of curiosity, is this change needed because we now work with struct returned by `sourceMapStore.transformCoverage`?

Reply:
yes @SergeAstapov